src/openrct2-ui/windows/Map.cpp

```diff
@@ -54,22 +54,22 @@ namespace OpenRCT2::Ui::Windows
     {
         return MapColour2((colour & 0xFF00) >> 8, PALETTE_INDEX_10);
     }
-    static int32_t getTechnicalMapSize()
+    static int32_t getPracticalMapSize()
     {
         // Take non-square maps into account
-        return std::max(GetGameState().MapSize.x, GetGameState().MapSize.y) - 2;
+        return std::max(std::max(GetGameState().MapSize.x, GetGameState().MapSize.y) - 2, 0);
```

Comment:
```suggestion
        return std::max({GetGameState().MapSize.x, GetGameState().MapSize.y, 0});
```
(Might not match what clang-format expects)

Reply:
`clang-format` has not complained, so it’s apparently fine with him.

I’d have to test to see if 1×1 would work, assuming stuff is going to end in tears :')

Reply:
> `clang-format` has not complained, so it’s apparently fine with him.

Heh, I meant my suggestion, not what's currently there. (I suspect `clang-format` would like spaces around the braces.)

Reply:
I think you removed the `- 2` in your suggestion by accident.

Reply:
Nice catch. Yes, that was unintentional.